lib/web/app/transport.go

```diff
@@ -159,39 +166,62 @@ func (t *transport) rewriteRequest(r *http.Request) error {
 	return nil
 }
 
-// websocketsDialer returns a function that dials a websocket connection
-// over the transport's reverse tunnel.
-func websocketsDialer(tr *http.Transport) forward.Dialer {
-	return func(network, address string) (net.Conn, error) {
-		conn, err := tr.DialContext(context.Background(), network, address)
+// DialContext
+func (t *transport) DialContext(ctx context.Context, _ string, _ string) (net.Conn, error) {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+
+	for i := len(t.c.servers) - 1; i >= 0; i-- {
```

Comment:
In this approach in scenario where all app agents are available the we will forward all connection to only one app agent without any LB approach.

Reply:
True, I'll add a shuffle on the `Match` function, so we get a new ordination of the server's list every time a new session is created and keep the current behavior.

Reply:
I would shuffle here as well, to avoid relying on some external behavior.